code/datums/diseases/advance/symptoms/emp.dm

```diff
@@ -0,0 +1,47 @@
+/datum/symptom/emp
+	name = "Electromagnetic Pulse"
+	desc = "The virus occasionally emits an electromagnetic pulse, disabling electronics in the host's inventory and on their tile."
```

Comment:
keep in mind anyone with a prosthetic limb hit by an EMP receives 10 burn and is stunned for 20 seconds each time.

Reply:
I think that's fine, it will be deadly if you have a prosthetic so either get cured fast or hack your limb off

Reply:
^ What PK said

Reply:
you can't because you're stunned 😏

Reply:
Too bad for you then